starter-code/app.js

```diff
@@ -1,5 +1,57 @@
-const express = require('express');
+const express = require("express");
+const expressLayouts = require("express-ejs-layouts");
 const app = express();
-const Chuck  = require('chucknorris-io');
+const Chuck = require("chucknorris-io");
 const client = new Chuck();
 
+app.use(expressLayouts);
+app.set("layout", "layouts/main-layout");
+
+app.set("views", __dirname + "/views");
+app.set("view engine", "ejs");
+
+app.use(express.static("public"));
+
+app.get("/random", (req, res, next) => {
+  client
+    .getRandomJoke()
+    .then(result => {
+      const data = {
+        joke: result
+      };
+      res.render("random", data);
+    })
+    .catch(err => {
+      throw err;
+    });
+});
+
+app.get("/random", (req, res, next) => {
```

Comment:
Why do you have the exact same route twice? The second one is never going to be reached.